src/app/Instance/InstancePage/ProcessorsTabContent.tsx

```diff
@@ -217,37 +196,36 @@ export const ProcessorsTabContent = ({
   return (
     <>
       <PageSection isFilled>
-        {processorListResponse?.items ? (
-          <TableWithPagination
-            columns={processorsOverviewColumns}
-            customToolbarElement={customToolbarElement}
-            rows={processorListResponse.items}
-            tableLabel={t(
-              "smartEventsTempDictionary:processor.processorsListTable"
-            )}
-            getRowOuiaId={rowOuiaId}
-            isLoading={areProcessorsLoading}
-            onPaginationChange={onPaginationChange}
-            pageNumber={currentPage}
-            pageSize={currentPageSize}
-            totalRows={totalRows ?? 0}
-            renderActions={renderActions}
-          >
-            <EmptyState variant="large">
-              <EmptyStateIcon icon={PlusCircleIcon} />
-              <Title headingLevel="h2" size="lg">
-                {t("processor.noProcessors")}
-              </Title>
-            </EmptyState>
-          </TableWithPagination>
-        ) : (
-          <TableWithPaginationSkeleton
-            hasActionColumn={true}
-            columns={processorsOverviewColumns}
-            totalRows={currentPageSize}
-            customToolbarElement={customToolbarElement}
-          />
-        )}
+        <TableView
+          ariaLabel={t(
+            "smartEventsTempDictionary:processor.processorsListTable"
+          )}
+          actions={[
+            /** TODO https://issues.redhat.com/browse/MGDOBR-1274 */
+            {
+              label: t("processor.createProcessor"),
+              onClick: onCreateProcessor,
+              isPrimary: true,
```

Comment:
There is not possible to add ouiaId. The action should be the create button in toolbar.

Reply:
I confirm. It is not currently possible to pass an ouiaId property to the Action configuration of the table.